src/llmq/quorums_chainlocks.cpp

```diff
@@ -337,31 +329,49 @@ void CChainLocksHandler::TrySignChainTip()
 
 void CChainLocksHandler::NewPoWValidBlock(const CBlockIndex* pindex, const std::shared_ptr<const CBlock>& block)
 {
-    LOCK(cs);
-    if (blockTxs.count(pindex->GetBlockHash())) {
-        // should actually not happen (blocks are only written once to disk and this is when NewPoWValidBlock is called)
-        // but be extra safe here in case this behaviour changes.
-        return;
-    }
+    // NotifyChainLock might not have been called yet, so call it now if needed.
+    bool fEnforceAndNotify = false;
 
-    int64_t curTime = GetAdjustedTime();
+    {
+        LOCK(cs);
 
-    // We listen for NewPoWValidBlock so that we can collect all TX ids of all included TXs of newly received blocks
-    // We need this information later when we try to sign a new tip, so that we can determine if all included TXs are
-    // safe.
+        if (blockTxs.count(pindex->GetBlockHash())) {
```

Comment:
Similar integer to boolean conversion.
`!= 0`

Reply:
Unlike in the previous case, this code wasn't touched (if you exclude whitespaces https://github.com/dashpay/dash/pull/2762/files?w=1) :P